app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -30,61 +32,67 @@ class ProjectBoard extends React.Component {
-            <AddStoryButton
-              onAdd={() => createStory({
-                state: Story.status.UNSCHEDULED
-              })}
-            />
-          }
-        >
-          <Stories stories={this.props.chillyBinStories} />
-        </Column>
-
-        <Column
-          title={`${I18n.t("projects.show.backlog")} /
-          ${I18n.t("projects.show.in_progress")}`}
-          renderAction={() =>
-            <AddStoryButton
-              onAdd={() => createStory({
-                state: Story.status.UNSTARTED
-              })}
-            />}
-        >
-          <Sprints
-            sprints={this.props.backlogSprints}
+      <DndProvider backend={HTML5Backend}>
+        <div className="ProjectBoard">
+          <Notifications
+            notifications={notifications}
+            onRemove={removeNotification}
           />
-        </Column>
 
-        <Column
-          title={I18n.t("projects.show.done")}
-        >
-          <Sprints
-            sprints={this.props.doneSprints}
-            fetchStories={this.props.fetchPastStories}
-          />
-        </Column>
+          <Column title={I18n.t("projects.show.chilly_bin")}
+            column="chillyBin"
+            renderAction={() =>
+              <AddStoryButton
+                onAdd={() => createStory({
+                  state: Story.status.UNSCHEDULED
+                })}
+              />
+            }
+          >
+            <Stories column="chillyBin" stories={this.props.chillyBinStories} />
+          </Column>
 
-        {
-          history.status !== 'DISABLED' &&
           <Column
-            onClose={closeHistory}
-            title={[I18n.t("projects.show.history"), "'", history.storyTitle, "'"].join(' ')}
+            title={`${I18n.t("projects.show.backlog")} /
+            ${I18n.t("projects.show.in_progress")}`}
+            column="backlog"
+            renderAction={() =>
+              <AddStoryButton
+                onAdd={() => createStory({
+                  state: Story.status.UNSTARTED
+                })}
+              />}
           >
-            { history.status === 'LOADED'
-              ? <History history={history.activities} />
-              : <div className="loading">Loading...</div>
-            }
+            <Sprints
+              column="backlog"
+              sprints={this.props.backlogSprints}
+            />
           </Column>
-        }
-      </div>
+
+          <Column
+            title={I18n.t("projects.show.done")}
+            column="done"
+          >
+            <Sprints
+              sprints={this.props.doneSprints}
+              fetchStories={this.props.fetchPastStories}
+            />
+          </Column>
+
+          {
+            history.status !== 'DISABLED' &&
+            <Column
+              onClose={closeHistory}
+              title={[I18n.t("projects.show.history"), "'", history.storyTitle, "'"].join(' ')}
+            >
+              { history.status === 'LOADED'
```

Comment:
You could create a function to check that and put in some _utils_ file. Ex:

export const isHistoryLoaded = ({ status }) =>  status === 'LOADED';
And use it here like `isHistoryLoaded(history)`